packages/botonic-plugin-contentful/src/contentful/cms-contentful.ts

```diff
@@ -236,7 +236,10 @@ export class Contentful implements cms.CMS {
   }
 
   async content(id: string, context = DEFAULT_CONTEXT): Promise<Content> {
-    const entry = await this._delivery.getEntry(id, context)
+    const REFERENCES_TO_INCLUDE = 6
+    const entry = await this._delivery.getEntry(id, context, {
+      include: REFERENCES_TO_INCLUDE,
+    })
```

Comment:
I think this parameter should be configurable so maybe the best option would be to pass this parameter as an input parameter for the `content` function making `6` as a default value.

Reply:
Yes, better this way, changed!